client/gutenberg/extensions/classic-block/edit.jsx

```diff
@@ -23,22 +23,56 @@ export class ClassicEdit extends Component {
 	}
 
 	storeEditor = ref => {
+		//By the time editor lifecycle events like PostProcess fire, this editor instance is already marked for removal
+		//so any content manipulation doesn't get reflected in this block.
+		//
+		//For now, when we detect an unmount, get content from the editor marked to be destroyed and update block attributes.
+		//https://github.com/Automattic/wp-calypso/issues/29643
+		if ( ! ref && this.editor ) {
```

Comment:
to be more explicit it could help to name this value. I think it doesn't take much time to figure out but that's partially because I know what to look for from the PR description…

```js
const isUnmounting = ! ref && this.editor;

if ( isUnmounting ) {
	…
}
```

Reply:
Good call. Added in 052ffe1